addons/hr/static/src/components/avatar_card/avatar_card_popover_patch.xml

```diff
@@ -4,6 +4,12 @@
         <xpath expr="//div[hasclass('o_avatar_card_buttons')]" position="inside">
             <button class="btn btn-secondary btn-sm" t-if="user.employee_ids?.length > 0" t-on-click.stop="onClickViewEmployee">View profile</button>
         </xpath>
+        <xpath expr="//a[hasclass('o_card_user_infos_tel')]" position="after">
+            <span class="text-muted" t-if="name_work_location_display" data-tooltip="Work Location">
+                <i class="fa-fw me-1" t-att-class="locationIcon"/>
```

Comment:
Ideally "fa" parts shouldn't be split from each others, as it makes maintenance (and potential upgrade of fa version) harder to do. The full "chain of modifiers" should be considered a single icon representation.